extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -525,6 +526,37 @@ Property values are stored in a compact binary tabular format described in the [
 
 As a result, byte length of the `BIN` chunk may be up to 7 bytes larger than JSON-defined `buffer.byteLength` to satisfy alignment requirements.
 
+## Assigning Metadata
+
+*Defined in [glTFChildOfRootProperty.EXT_structural_metadata.schema.json](./schema/glTFChildOfRootProperty.EXT_structural_metadata.schema.json).*
+
+When property values are stored in a [Property Table](#property-tables), then the entries of this table may be associated with elements of the glTF asset. This applies to all elements that are a `glTFChildOfRootProperty`. Each of these elements can contain an `EXT_structural_metadata` object that defines the source of the metadata values for this element. It contains the `propertyTable`, which is the index of the property table in the array of property tables of the root-level `EXT_structural_metadata` extension object, and the `index`, which is the index of the row in this table that contains the metadata values for the respective element.
```

Comment:
> This applies to all elements that are a `glTFChildOfRootProperty`. 

`glTFChildOfRootProperty` feels a bit too jargony unless someone is familiar with the glTF schema. Should the types be listed out explicitly like in [KHR_xmp_json_ld](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_xmp_json_ld#overview)? Note that some types like `accessor` and `bufferView` are omitted. As a side note, which approach is easier for code generators?

If we list them explicitly maybe something like this:

> Property table values can be referenced from glTF objects of type: (list out types explicitly). Each of these elements can contain an `EXT_structural_metadata` object that defines the source of the metadata values for this element. It contains the `propertyTable`, which is the index of the property table in the array of property tables of the root-level `EXT_structural_metadata` extension object, and the `index`, which is the index of the row in this table that contains the metadata values for the respective element.